client/src/components/PacmanGame/Ghosts.jsx

```diff
@@ -1,15 +1,21 @@
 import React from 'react';
 import Cell from './Cell';
 
-const Ghosts = ({ ghosts, gridSize }) => ghosts.map(({ x, y }, index) => {
+const Ghosts = ({ ghosts, gridSize }) => ghosts.map(({ x, y, direction }, index) => {
+  let entity;
```

Comment:
This can be something like this: 

const entity = !direction ? 'scatterGhost' : 'ghost'